src/main/resources/view/MainWindow2.fxml

```diff
@@ -13,48 +13,48 @@
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
          title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
-  <icons>
-    <Image url="@/images/GoodMatch_logo.png" />
-  </icons>
-  <scene>
-    <Scene>
-      <stylesheets>
-        <URL value="@DarkTheme.css" />
-        <URL value="@Extensions.css" />
-      </stylesheets>
+    <icons>
```

Comment:
will these tabs break anything?

Reply:
the tabs do not break anything, checkstyle and gradle clean build run correctly

Reply:
The tabs were added due to intellij's auto indent for code style, they do not break anything.
Gradle clean build runs fine and the app is working normally

Reply:
fixed